### PR TITLE
inputs.x509_cert: Fix timeout issue 

### LINKED ISSUE #8809: telegraf plugin x509_cert stopped working from 1.17.1 upwards with dial tcp: i/o timeout
### Relevant telegraf.conf:

```toml

```

### System info:

1.17.1 + 1.17.2

### Steps to reproduce:

1. Use x509_cert Plugin to verify SSL Certs
2. Run telegraf with --test

### Expected behavior:

telegraf retrieves ssl version information and performs a chain check:

```
x@odo:/etc/telegraf/telegraf.d# sudo telegraf --test --config certs.conf
2021-02-04T23:20:56Z I! Starting Telegraf 1.17.0
> x509_cert,common_name=testhost.net,host=odo,issuer_common_name=R3,public_key_algorithm=RSA,san=testhost.net,serial_number=40d124535dbdeb52a71517554264ddcb576,signature_algorithm=SHA256-RSA,source=https://testhost.net:443,verification=valid age=2271661i,enddate=1617985195i,expiry=5504338i,startdate=1610209195i,verification_code=0i 1612480857000000000
```

### Actual behavior:

plugin fails with a dial tcp: i/o timeout error:

```x@odo:/etc/telegraf/telegraf.d# sudo telegraf --test --config certs.conf
2021-02-04T23:20:12Z I! Starting Telegraf 1.17.1
2021-02-04T23:20:12Z E! [inputs.x509_cert] Error in plugin: cannot get SSL cert 'https://testhost.net:443': dial tcp: i/o timeout
```

### Additional info:

Config:

```
[[inputs.x509_cert]]
  interval = "300s"

  ## List certificate sources
  sources = [
        "https://testhost.net:443"
  ]
```

Stopped working with 1.17.1 - after downgrade to 1.17.0 it's working again.

### PR DESCRIPTION
**Pull request by viperstars**
closes #8809

If timeout is not configured, it will be set to "5ns" when adding to inputs. This cause the timeout issue #8809. Test case for reproducing is added in commit ac4cda1.

I fix the issue by setting timeout to "0s".

### CODE REVIEW

File: plugins/inputs/x509_cert/x509_cert.go [hunk 1]
```diff
@@ -280,7 +280,6 @@ func init() {
 	inputs.Add("x509_cert", func() telegraf.Input {
 		return &X509Cert{
 			Sources: []string{},
-			Timeout: internal.Duration{Duration: 5},
```
Comment (srebhan) [inline plugins/inputs/x509_cert/x509_cert.go:283]: This is not the correct fix. You should instead initialize the duration correctly as `5*time.Second`.

**Review by srebhan — APPROVED** [review]
Thanks for fixing this!

**Comment by helenosheaa** [comment]
@viperstars looks good. Can you please add closes #8809 to your description so it will close the issue when we merge?

**Comment by viperstars** [comment]
> @viperstars looks good. Can you please add closes #8809 to your description so it will close the issue when we merge?

It is added in the new commit message now.

**Comment by srebhan** [comment]
@viperstars I think github only evaluates the final commit message which is the PR description above. That's due to the fact that the changes are squashed before merge. So please simply open the PR at the github site and click `edit` in the top PR message. Then add the line `closes #8809` and save. There is no need for a code change.

**Comment by viperstars** [comment]
> @viperstars I think github only evaluates the final commit message which is the PR description above. That's due to the fact that the changes are squashed before merge. So please simply open the PR at the github site and click `edit` in the top PR message. Then add the line `closes #8809` and save. There is no need for a code change.

It is updated now.

**Review by helenosheaa — APPROVED** [review]

**Review by srebhan — APPROVED** [review]
Perfect.

**Review by Jagularr — APPROVED** [review]

**Pull request merged**